README.md

```diff
@@ -4,17 +4,20 @@ Point cloud rendering on mobile using Euclideon's udSDK.
 
 ## Getting Started
 
-This project is a starting point for a Flutter application.
+Install flutter -> [instructions](https://flutter.dev/docs/get-started/install)
 
-A few resources to get you started if this is your first Flutter project:
+Download Euclideon's [udSDK](https://www.euclideon.com/udsdk/)
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+Copy all header files from `udkSDK/include` into `vaultSDK/ios/Classes/`
```

Comment:
Shouldn't this be part of the iOS steps? Or do you use this path for the binding generation?

Reply:
We use this path for binding since 
> "CocoaPods doesn’t allow including sources above the podspec file, but Gradle allows you to point to the ios folder"
https://flutter.dev/docs/development/platform-integration/c-interop#step-2-add-cc-sources

right @kressnick25 ?

Reply:
Yu is correct. However I just checked and the app works fine without these header files at all. I'll remove this step from the setup process.